packages/data-stores/src/plans/constants.ts

```diff
@@ -17,18 +22,45 @@ interface Currency {
 	decimal: number;
 }
 
-export const plansOrder = [ PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS, PLAN_ECOMMERCE ];
+export const plansOrder = [
+	PLAN_PERSONAL,
+	PLAN_PREMIUM,
+	PLAN_BUSINESS,
+	PLAN_ECOMMERCE,
+	PLAN_PERSONAL_MONTHLY,
+	PLAN_PREMIUM_MONTHLY,
+	PLAN_BUSINESS_MONTHLY,
+	PLAN_ECOMMERCE_MONTHLY,
+];
```

Comment:
Just a suggestion to reduce repetition and make things a bit easier to grasp.

If `plansOrder` is declared after `billedYearlySlugs` and `billedMonthlySlugs`, it could be re-written like this:

```suggestion
export const plansOrder = [ ... billedYearlySlugs, ...billedMonthlySlugs ];
```